### PR TITLE
Fixed struct formatting. Closing issue #5032.

### PR DESCRIPTION
**Pull request by orizi**
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/cairo/5062)

### CODE REVIEW

File: corelib/Scarb.lock
Comment (delaaxe) [inline corelib/Scarb.lock:1]: in this file intented?

**Review by orizi — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/cairo/5062)* status: 0 of 9 files reviewed, 1 unresolved discussion (waiting on @delaaxe and @gilbens-starkware)

___
*[`corelib/Scarb.lock` line 0 at r1](https://reviewable.io/reviews/starkware-libs/cairo/5062#gh-1488744379:-NqaSqt52Af9LMmgJNYi:b-896fix) ([raw file](https://github.com/starkware-libs/cairo/blob/e74f5214876af0dbecfbee474615b7c80d4aa8ab/corelib/Scarb.lock#L0)):*
<details><summary><i>Previously, delaaxe wrote…</i></summary><blockquote>

in this file intented?
</blockquote></details>

Done.

**Review by gilbens-starkware — APPROVED** [review]
:lgtm:

> Reviewed 8 of 9 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/cairo/5062#-:-NqazjTI2-wP5qCZAvnP:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @delaaxe)

**Review by orizi — COMMENTED** [review]
> Reviewed 1 of 1 files at r2, 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/cairo/5062)* status: all files reviewed (commit messages unreviewed), 1 unresolved discussion (waiting on @delaaxe)

**Pull request merged**